locale/meson.build

```diff
@@ -0,0 +1,24 @@
+fs = import('fs')
+
+mo_targets = []
+
+po_files_raw = run_command('find', '.', '-name', '*.po', check : true).stdout().strip().split('\n')
+po_files = files(po_files_raw)
+
+foreach po_file : po_files
+  language = fs.name(fs.parent(fs.parent(po_file)))
```

Comment:
I would like to see 4-space indentations instead of 2, for better readability and to match surrounding meson code.

Reply:
OK